### ISSUE TITLE
`BI3`: Correggere errori pa11y

### ISSUE THREAD

**Issue by Fupete** [italia/bootstrap-italia#1650]
### Esiste già una discussione sul tema che ti interessa, o su un tema simile?

- [x] Ho verificato e non esiste

### Versione della libreria

3.x

### Cosa

Lanciando la CI di pa11y segnala questi problemi: 

## Errori di accessibilità rilevati da pa11y

- **`/docs/componenti/chips/`** - Contrasto insufficiente (3.7:1 invece di 4.5:1) su 4 elementi `<span class="chip-label">Etichetta</span>` - Raccomandazione: cambiare colore testo in `#4e6174` **<< Dipendono dallo sfondo grigino della documentazione e dal fatto che la Chip non ha fondo bianco ma è trasparente.** 

- **`/docs/menu-di-navigazione/footer/`** - Input `#exampleFormGroup` senza label accessibile: `<input type="text" class="form-control form-control-sm" id="exampleFormGroup" placeholder="Inserisci la tua email">` - Aggiungere `<label>`, `title`, `aria-label` o `aria-labelledby`

- **`/docs/menu-di-navigazione/footer/`** - ID duplicato `exampleFormGroup` (2 istanze)

- **`/docs/esempi/form/`** - ID duplicato `etichetta_input_group` su `<input type="text" class="form-control" id="etichetta_input_group" name="etichetta_input_group" placeholder="Lorem Ipsum">`

- **`/docs/esempi/form/`** - ID duplicato `nome` su `<input type="text" class="form-control" id="nome" placeholder="Inserisci un testo" required="" disabled="">`

- **`/docs/esempi/liste/`** - ID duplicato `checkbox6` (2 istanze) su `<input type="checkbox" id="checkbox6" checked="">`

- **`/docs/form/input/`** - Input disabilitato `#input-text-disabled` senza label accessibile: `<input class="form-control" type="text" id="input-text-disabled" disabled="">` - Aggiungere `<label>`, `title`, `aria-label` o `aria-labelledby`

- **`/docs/form/input/`** - ID duplicati: `input-group-1`, `input-group-2`, `input-group-3`, `button-3`

- **`/docs/componenti/sections/`** - Contrasto critico (1.09:1) su `<h2 id="titleEx4" class="no_toc">Morbi fermentum amet</h2>` - Raccomandazione: cambiare sfondo in `#949494`

- **`/docs/componenti/sections/`** - Contrasto critico (1.09:1) su 3 paragrafi nella sezione - Raccomandazione: cambiare sfondo in `#767676`

### Perché

Importante controllare segnalazioni automatiche a11y. 
cc @astagi @zetareticoli

**Comment by Fupete** [comment]
Cose del genere possiamo ignorare. Disabled va bene con contrasto minore di 4.5:1. E quel Js è ovviamente un esempio. 

In futuro possiamo segnarci di integrare la chiamata a pa11y nella CI con regole per ignorare elementi specifici, dovrebbe essere questo il modo: https://github.com/pa11y/pa11y#hideelements-string.
